.vale/styles/RedHat/ConfigMap.yml

```diff
@@ -1,8 +0,0 @@
----
-extends: existence
-ignorecase: true
-level: suggestion
-link: https://redhat-documentation.github.io/vale-at-red-hat/docs/reference-guide/configmap/
-message: "Do not use 'config', unless it is followed by 'map'."
```

Comment:
This message is obsolete. 
Do we still need to provide users with some guidance to replace configmap and configuration map based on https://redhat-documentation.github.io/supplementary-style-guide/#config-map ?
Or, will the new user experience provide enough information?